src/renderer/components/ApplicationStepsDropdown.tsx

```diff
@@ -0,0 +1,150 @@
+import { useState, useRef, useEffect } from 'react'
+import './ApplicationStepsDropdown.css'
+import { ApplicationStep } from '../types'
+
+interface ApplicationStepsDropdownProps {
+  steps: ApplicationStep[]
+  onUpdateSteps: (steps: ApplicationStep[]) => void
+}
+
+const DEFAULT_STEPS = [
+  'Resume submitted',
+  'Cover letter written',
+  'Application submitted',
+  'Follow-up sent',
+  'Phone screen scheduled',
+  'Interview scheduled',
+  'Thank you note sent'
+]
```

Comment:
```suggestion
const DEFAULT_STEPS = [
  'Resume tailored',
  'Cover letter written',
  'Application submitted',
  'Follow-up sent',
  'Phone screen scheduled',
  'Interview scheduled',
  'Thank you note sent'
]
```
@loveanimals4ever please provide input